### ISSUE TITLE
Add support for VST3 Note Expressions

### ISSUE THREAD

**Issue by kurasu** [surge-synthesizer/surge#11]
Should allow for per-voice control of

- Pitch
- Timbre
- Pressure
- Volume
- Pan

Compatible hosts (at the moment) would be Cubase and Bitwig.

**Comment by baconpaul** [comment]
At this point we are waiting for if and when juce supports this since I dumped our hand rolled vst3

**Comment by mkruselj** [comment]
We should leave the issue open so that we don't forget about it if it happens in JUCE...

**Comment by mkruselj** [comment]
It looks like this will become a 1.x feature due to CLAP wrapper, so moving it from 2.0 milestone to 1.x milestone.

**Comment by baconpaul** [comment]
Is there a daw which supports vst3 note expressions, doesn’t support clap, and isn’t cubase?

**Comment by mkruselj** [comment]
I'm not sure if there are any, actually.

**Comment by baconpaul** [comment]
Right. So this really is 'make note expressions work in cubase'. That's all I was checking.

**Comment by mkruselj** [comment]
Until eventually some other hosts support it, who knows, yeah.

**Comment by kurasu** [comment]
I guess Steinberg (or some Cubase user) can implement it themselves if they find it important?

**Comment by mkruselj** [comment]
Or (more likely) we delegate it to CLAP wrapper at some point and then it produces a VST3 that will have note expressions enabled because Timo did an awesome work.

**Comment by baconpaul** [comment]
Yeh it works in cubase via the clap wrapper claes - we just need to finish the wrapper

**Comment by mkruselj** [comment]
Link to #7203.
